lib/pharos/phases/validate_host.rb

```diff
@@ -74,6 +75,19 @@ def cpu_arch
           id: cpu['Architecture']
         )
       end
+
+      # @param interface [String]
+      # @return [String]
+      def private_interface_address(interface)
+        @ssh.exec!("ip -o addr show dev #{interface} scope global | awk '{ print $4 }'").each_line do |line|
+          ip, prefixlen = line.split('/')
+
+          next if ip == @host.address
```

Comment:
Show allow this to work on the likes of `packet.net`, which configure both the public IP and private IP on the same `bond0` interface.